dluhc-component-library/src/components/siteSelectionForm/utils.ts

```diff
@@ -69,3 +71,15 @@ export const createValidationSchema = (
 
   return object({ [key]: validationShape } as ObjectShape);
 };
+
+export const storeFormData = (formData: FormState) => {
+  return db.formDataStore.put({ id: "siteSelectionForm", data: formData });
```

Comment:
We could maybe pull id out into a `const` since the same string is used here and in the get function below